src/content/docs/new-relic-solutions/new-relic-solutions/optimize-cloud-architecture-spend-continuously-improve-your-modern-cloud-environment.mdx

```diff
@@ -147,7 +147,7 @@ In the sample dashboard above, the CPU-usage widget on the left reveals that thi
 
 This is just one example of how to identify cloud-based resources that could be candidates for optimization.
 
-Now that you've identified architecture for optimization, go ahead and do so. Whether you right-size your instances, fine tune your databases, modify your storage usage, better configure your load balancers, or even re-architect your applications, in the end your goal is to be able to compare your new, optimized architecture against the baselines you captured in Step 2. For more about baselines please review the [Establish Objectives & Baselines tutorial](/docs/1-adopt-easily-establish-objectives-baselines-define-team-slos).
+Now that you've identified architecture for optimization, go ahead and do so. Whether you right-size your instances, fine tune your databases, modify your storage usage, better configure your load balancers, or even re-architect your applications, in the end your goal is to be able to compare your new, optimized architecture against the anomaly you captured in Step 2. For more about anomaly please review the [Establish Objectives & Anomaly tutorial](/docs/1-adopt-easily-establish-objectives-anomalys-define-team-slos).
```

Comment:
Should this be `For more about anomalies`?

Also, did this URL change?

Reply:
fixed!